pkg/volume/csi/csi_mounter.go

```diff
@@ -400,6 +400,7 @@ func (c *csiMountMgr) supportsFSGroup(fsType string, fsGroup *int64, driverPolic
 		klog.V(4).Info(log("mounter.SetupAt WARNING: skipping fsGroup, access modes not provided"))
 		return false
 	}
+	// TODO: Investigate if this needs updating to support ReadWriteOncePod.
```

Comment:
@jsafrane @msau42 

Michelle and I synced offline on this. It was suggested that we don't add this to include `ReadWriteOncePod`. If users want to leverage `FSGroupPolicy` with `ReadWriteOncePod` volumes, they will need to create a `CSIDriver` object and set the field. Any thoughts on this?

Reply:
ack